packages/fxa-graphql-api/src/test/lib/datasources/profileServer.spec.ts

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import 'mocha';
+import 'reflect-metadata';
```

Comment:
Oh and one other thing I'm scratching my head at: What does reflect-metadata do for us? Seems like a polyfill, saw google results that suggested tests on decorator-using code needed it. But I was curious and commented it out to see what happens, and the tests still pass.

Reply:
The Dependency Injection libraries like inversify and typedi (used here) require reflect-metadata to be present, before any of their code loads. Technically it only needs to happen once, so a root test runner could do it and none of these files would then need it. However, then we wouldn't be able to run the tests just for a single test file, like with the VS Code 'Mocha (Run current file)' launch task. Thus, I end up adding these to each spec file, cause I really like running just the file I'm working on.

Reply:
Ah yeah, that kind of thing belatedly occurred to me this morning... If we need more of this sort of thing, we could add a global [`test-setup.ts`](https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/test-setup.ts) that runs for all tests (even single runs)